### PR TITLE
Fix build break

### PR DESCRIPTION
**Pull request by smitpatel**
There are further tests being affected by #4458 breaking the build.
Converted Assert.Equal to Assert.Contains to verify SQL ignoring the ordering of statements

### CODE REVIEW

File: test/Microsoft.EntityFrameworkCore.SqlServer.FunctionalTests/QuerySqlServerTest.cs [hunk 1]
```diff
@@ -3363,25 +3386,25 @@ public override void Projection_when_arithmetic_expression_precendence()
...
-//        public override void Projection_when_arithmetic_mixed()
-//        {
-//            //base.Projection_when_arithmetic_mixed();
-//        }
-//
-//        public override void Projection_when_arithmetic_mixed_subqueries()
-//        {
-//            //base.Projection_when_arithmetic_mixed_subqueries();
-//        }
+        //        public override void Projection_when_arithmetic_expressions()
```
Comment (divega) [inline test/Microsoft.EntityFrameworkCore.SqlServer.FunctionalTests/QuerySqlServerTest.cs]: Is this change intentional? Do you know why is it commented out?

**Comment by divega** [comment]
:shipit:

File: test/Microsoft.EntityFrameworkCore.SqlServer.FunctionalTests/QuerySqlServerTest.cs (hunk 1 above)
Comment (smitpatel) [inline test/Microsoft.EntityFrameworkCore.SqlServer.FunctionalTests/QuerySqlServerTest.cs]: It was already commented out. TODO note above it says complex projection translation.
Autoformat on file changed formatting. I will revert it so the commit does not mislead in future.

**Comment by maumar** [comment]
Interesting. I ran full build when working on LOJ translation and fixed all tests that were affected for me. Possibly the issue is intermittent?

**Comment by smitpatel** [comment]
Doesn't look like intermittent. Once I have seen failure, it always failed for future runs. (also check CI history).

**Comment by divega** [comment]
@smitpatel @maumar How hard would it be to get a minimal repro of the behavior difference? I would like us to pass this along as an issue for the CoreCLR team. I know Maurycy started but not sure how far along he got.
